source/icu.net/SafeEnumeratorHandle.cs

```diff
@@ -23,9 +21,7 @@ public SafeEnumeratorHandle() : base(IntPtr.Zero, true)
 		/// failure, false. In this case, it generates a ReleaseHandleFailed Managed Debugging
 		/// Assistant.
 		///</returns>
-#if !NETSTANDARD1_6 && !NET
```

Comment:
This shows deprecation warnings on GitHub, so I think we still need this line, and we can't remove it because we still support net40 and net451. But it might be easier to parse for the reader if we change it to

```
#if NETFRAMEWORK
```

Reply:
Ah ok I didn't notice that. Yeah I originally rewrote it to that, but then went ahead and removed it. I'll go through and update this and similar lines.